lib/circuitry/publisher.rb

```
@@ -52,7 +51,7 @@ def publish_internal(topic_name, message)
         Timeout.timeout(timeout) do
           logger.info("Publishing message to #{topic_name}")
 
-          topic = TopicCreator.find_or_create(topic_name)
+          topic = Topic.find(topic_name)
```

Should we raise an error here if the topic isn't found?